MarkdownView/markdownview/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="us.feras.mdv">
-
-    <application
-        android:allowBackup="true"
-        android:label="@string/app_name"
-        android:supportsRtl="true">
```

Comment:
At some point, I was thinking of adding different processors options for Markdown. So you could use from one of the different predefined options like MarkdownJ and maybe you could also supply your own processor. Let me know if that make sense, I think it could be a good feature to add.